modules/ossm-federation-config-smcp.adoc

```diff
@@ -0,0 +1,397 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-config-smcp_{context}"]
+= Configuring a control plane for federation
+
+Before a mesh can be federated, you must configure the `ServiceMeshControlPlane` for mesh federation.   Because  all meshes that are members of the federation are equal, and each mesh is managed independently,  you must configure the SMCP for _each_ mesh that will participate in the federation.
+
+In the following example, the administrator for the `red-mesh` is configuring the SMCP for federation with both the `green-mesh` and the `blue-mesh`.
+
+.Sample SMCP for red-mesh
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: red-mesh
+  namespace: red-istio-system
+spec:
+  version: v2.1
+  cluster:
+    name: red-cluster
+    network: red-network
```

Comment:
@rcernich do we need to mention .spec.cluster.network in federation SMCP CR ?

Reply:
Neither of these is required.

Reply:
So should lines 24 and 25 be deleted?

Reply:
yep. the whole cluster block (l23-l25) can be removed

Reply:
deleted